### PR TITLE
DIV 3967 landing page for  corespondent 

### PR DESCRIPTION
**Pull request by mounikahmcts**
https://tools.hmcts.net/jira/browse/DIV-3967

Moved existing steps to respondent folder and creating correspondent related steps in correspondent folder.

As a Co-respondent
I want to be able to start entering my response online
So that I can submit my response to the application I have been named in.

https://tools.hmcts.net/jira/browse/DIV-3987

As a Co-respondent

I want confirm that I intend to defend the divorce
So that I can influence whether the application will progress.
and

https://tools.hmcts.net/jira/browse/DIV-3988

I want to indicate whether I am willing to pay any costs relating to the divorce
So that this can be taken into account when the Court decides who has to pay.

https://tools.hmcts.net/jira/browse/DIV-3974

As a Co-respondent
I want review the application for divorce that has been submitted naming me in an allegation of Adultery
So that I can fully consider what my response will be.

https://tools.hmcts.net/jira/browse/DIV-3978
As a Co-respondent
I want indicate whether I admit to the claim of adultery
So that I can provide by opinion on the divorce application details.

https://tools.hmcts.net/jira/browse/DIV-3980
As a Co-respondent
I want indicate whether I will seek to defend the divorce
So that I can influence whether the application will progress.

### CODE REVIEW

File: steps/correspondent/cr-respond/CrRespond.content.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "continue": "Continue",
+    "yourCourt": "Your divorce centre",
+    "email": "Email",
+    "gettingHelp": "Getting help",
+    "careOfAbbreviation": "c/o",
+    "phone": "Phone",
+    "divorceCenterHours": "Monday to Friday, 8.30am to 5pm",
+    "guidance": {
+      "title": "Guidance on GOV.UK",
+      "howWifeHusbandResponds": "<a href=\"https://www.gov.uk/respond-divorce-application\" target=\"_blank\" aria-label=\"This link will open in a new tab.\">Responding to a divorce application</a>",
```
Comment (hosainnet) [inline steps/correspondent/cr-respond/CrRespond.content.json]: Not sure how easy it is to put these in config?

File: mocks/steps/idamLogin/IdamLogin.content.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
         "yesCaseProgressedDefending": "Yes - case progressed, defended & answered",
         "no": "No",
         "petitionerRedirect": "Set user to petitioner",
+        "correspondentLogin": "Set user to Correspondent",
```
Comment (tchow8) [inline mocks/steps/idamLogin/IdamLogin.content.json]: Should be Co-Respondent.

Comment (tchow8) [reply mocks/steps/idamLogin/IdamLogin.content.json]: Remember to fix everywhere with 2 r's, correspondent is different from co-respondent.

File: config/default.yml [hunk 3]
```diff
@@ -67,33 +67,36 @@ services:
...
+    legalProceedings: /legal-proceedings
+    agreeToPayCosts: /agree-to-pay-costs
+    contactDetails: /contact-details
+    checkYourAnswers: /check-your-answers
+    invalidCaseState: /invalid-case-state
+    noConsentAreYouSure: /no-consent-are-you-sure
+    progressBar: /progress-bar
+    done: /done
+  correspondent:
+    respond: /corespondent-aos/progress-bar/respond
```
[the remark this answers is not in the document]
Comment (mounikahmcts) [reply config/default.yml]: > It should be `co-respondent`, and maybe we don't need `-aos`, as we're already in the AOS app?

Url details are provided by Neil in the prototype

File: mocks/steps/idamLogin/IdamLogin.content.json (hunk 2 above)
Comment (mounikahmcts) [reply mocks/steps/idamLogin/IdamLogin.content.json]: fixed 2r's

File: config/default.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (hosainnet) [reply config/default.yml]: We can have a chat with Neil, I'm not sure it's a strict requirement to have `aos` in the URL

[the remark this answers is not in the document]
Comment (mounikahmcts) [reply config/default.yml]: Moved urls to co-respondent and the folder structure

File: middleware/redirectMiddleware.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 const config = require('config');
 
+const crRespond = require('steps/correspondent/cr-respond/CrRespond.step');
+
```
[the remark this answers is not in the document]
Comment (mounikahmcts) [reply middleware/redirectMiddleware.js]: Moved folder structure

[the remark this answers is not in the document]
Comment (hosainnet) [reply middleware/redirectMiddleware.js]: Thanks 👍

File: steps/co-respondent/cr-review-application/CrReviewApplication.step.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  get const() {
+    return values;
+  }
+
+  get session() {
+    return this.req.session;
+  }
+
+  get feesIssueApplication() {
+    return this.res.locals.applicationFee['petition-issue-fee'].amount;
```
Comment (hosainnet) [inline steps/co-respondent/cr-review-application/CrReviewApplication.step.js]: We have many of these magic fee type values duplicated in multiple files, might be good to put in a constants file (not too urgent)

File: steps/co-respondent/cr-agree-to-pay-costs/CrAgreeToPayCosts.content.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+ 
+    "question": "Do you agree to pay the costs?",
+    "notice": "A costs order will only cover the divorce costs, and won't include any costs related to financial settlements.",
+
+    "info": {
+      "heading": "Read more about costs orders",
+      "options": {
+        "amountToPay": {
+          "heading": "How much you may have to pay",
+          "summary1": "The applicant's costs usually include the £550 fee. They could also include other court fees and costs. The costs could be much higher if the applicant has legal fees (eg a solicitor) and these are also included.",
```
[the remark this answers is not in the document]
Comment (mounikahmcts) [reply steps/co-respondent/cr-agree-to-pay-costs/CrAgreeToPayCosts.content.json]: > Should the £550 fee be dynamic?

Updated dynamic fee

File: steps/co-respondent/cr-review-application/CrReviewApplication.step.js (hunk 5 above)
Comment (mounikahmcts) [reply steps/co-respondent/cr-review-application/CrReviewApplication.step.js]: > We have many of these magic fee type values duplicated in multiple files, might be good to put in a constants file (not too urgent)

Yes, we can move it to constants. Will do in a new . ticket

Comment (hosainnet) [reply steps/co-respondent/cr-review-application/CrReviewApplication.step.js]: 👍

**Pull request merged**